titan/exposures/hiv.py

```diff
@@ -326,11 +326,7 @@ def progress_to_aids(self, model: "model.TITAN"):
         """
         aids_params = self.agent.location.params.hiv.aids
         p = 1.0
-        if self.agent.haart.active:  # type: ignore[attr-defined]
-            if self.agent.haart.adherent:  # type: ignore[attr-defined]
-                p = aids_params.haart_scale.adherent
-            else:
-                p = aids_params.haart_scale.non_adherent
+        self.agent.haart.aids_scale()  # type: ignore[attr-defined]
```

Comment:
`aids_scale` returns a prob, but this isn't being used here, should line 328 be `p = self.agent.haart.aids_scale()`?